index.js

```diff
@@ -31,9 +31,22 @@ module.exports = {
   included(includer) {
     this._super.included.apply(this, arguments);
 
-    let buildTarget = includer.options &&
+    // If this is set, show a warning
+    let explicitBuildTarget = includer.options &&
       includer.options[this.name] &&
       includer.options[this.name]['buildTarget'];
+    if (explicitBuildTarget) {
+      this.ui.writeWarnLine('You no longer need to specify a buildTarget - it is now derived from your project files. Please delete this config option.')
+    }
+
+    let buildTarget;
+    if (fs.existsSync(includer.project.root + '/app/tailwind')) {
+      buildTarget = 'app';
+    } else if (fs.existsSync(includer.project.root + '/addon/tailwind')) {
```

Comment:
@samselikoff is the idea of the `addon` target that an addon can install `ember-cli-tailwind` and any consuming app will then get to use tailwind with the configuration provided by the addon?

If so, won't this be looking for `/addon/tailwind` in the consuming app which wont exist? Perhaps we should just default `buildTarget` to `addon` and check the existence of tailwind folders for `app` and `dummy`?